### PR TITLE
Add function to humanize strings

### PR DESCRIPTION
**Pull request by josiahsprague**
just start with humanizing the page's tag parameter

### CODE REVIEW

File: preprocessors/site.js [hunk 1]
```diff
@@ -22,6 +24,13 @@ module.exports = function(context) {
   return context;
 };
 
+var humanizePageData = function(context) {
+  if (context.parameters.tag) {
+    context.page.tag = humanize(context.parameters.tag);
```
Comment (joanniclaborde) [inline preprocessors/site.js]: You can't do that, the URL tag is used by widgets, to create links to other pages. Maybe create a new context key, like `tag_name`?

Comment (josiahsprague) [inline preprocessors/site.js]: Do widgets use `page.tag` or `parameters.tag`? I thought `page.tag` was new.

Comment (joanniclaborde) [inline preprocessors/site.js]: Doh, my mistake, I read it as `context.parameters.tag = humanize(context.parameters.tag);`. Move along, nothing to see here.

Comment (josiahsprague) [inline preprocessors/site.js]: :-P Sounds good!

**Pull request merged**